task3/src/services/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response } from 'express';
```

Comment:
Update service to make requests to the db. Don't use req/res here as this is the responsibility of router/controller

Reply:
Understood!